### ISSUE TITLE
Add docs for building images with composer

### ISSUE THREAD

**Issue by nullr0ute** [osbuild/images#143]
With the split out from composer it would be useful to have some docs on how to build this with composer to make it easier for people to build and test images.

**Comment by achilleas-k** [comment]
Our [HACKIND.md](https://github.com/osbuild/images/blob/main/HACKING.md) describes how to generate manifests and build images directly without osbuild-composer, which for most cases should be enough.

The [testing docs in osbuild-composer](https://github.com/osbuild/osbuild-composer/tree/main/test#image-tests) explain how to build and test osbuild-composer with a development version of osbuild/images.

I'll keep this issue open as a reminder to link those from the project README.  I expect most people looking for this kind of information would start there.

**Comment by nullr0ute** [comment]
I found the HACKING.md after I filed this, and hadn't found the other docs, a developing paragraph with links would definitely make things more discoverable.

**Comment by achilleas-k** [comment]
I should also add a brief description of how the RPM repositories work (snapshots, `test/data/repositories/`).  I expect sometimes people might want to change those (maybe to upstream or a different snapshot) when testing.

**Comment by teg** [comment]
@achilleas-k would it make sense to share your demo recording? That covers a lot I think.

**Comment by nullr0ute** [comment]
> @achilleas-k would it make sense to share your demo recording? That covers a lot I think.

It may be useful but I think it's better to be able to read docs, I for one won't got and watch a video.

**Comment by achilleas-k** [comment]
Yes, I think that would be very useful.  We would need to put it somewhere public though and I don't know who owns the recording.  @ochosi ?

**Comment by achilleas-k** [comment]
> > @achilleas-k would it make sense to share your demo recording? That covers a lot I think.
> 
> It may be useful but I think it's better to be able to read docs, I for one won't got and watch a video.

Not a huge fan of video tutorials myself either but I guess it works for some people.

**Comment by ochosi** [comment]
> Yes, I think that would be very useful.  We would need to put it somewhere public though and I don't know who owns the recording.  @ochosi ?

I guess it's possible to share it publicly, if you like. It ofc comes with the usual downsides of video tutorials (i.e. @nullr0ute doesn't watch them and they are difficult to update once things change 😉). Let me know if you want me to go ahead!

**Comment by achilleas-k** [comment]
Let's do it.  We can take it down, unlink it, or add a big outdated warning when things change enough.  Maybe I'll do another one by then :grin: 
Do we have a project YouTube channel? :)
